web-client/src/ducks/posts/functions.ts

```diff
@@ -1,15 +1,19 @@
+import { isDefined } from 'class-validator';
```

Comment:
Renamed functions, added query for getting posts based on location, which can be removed since we are now using Algolia.